packages/picker/test/index.ts

```diff
@@ -96,6 +96,9 @@ export function runPickerTests(): void {
             await expect(el).to.be.accessible();
         });
         it('closes accessibly', async () => {
+            el.focus();
+            await elementUpdated(el);
+            expect(el.shadowRoot?.activeElement).to.equal(el.button);
```

Comment:
Did TS require the `?` here? We should be force adding `shadowRoot` to or base class, so just making sure something isn't missing else where.

Reply:
Adding the optional chaining was just me. I'll remove it.